src/AddIns/Uno.UI.Lottie/LottieVisualSourceBase.cs

```diff
@@ -87,37 +87,51 @@ public Task SetSourceAsync(Uri sourceUri)
 #if !(__WASM__ || (__ANDROID__ && !NET6_0_OR_GREATER) || (__IOS__ && !NET6_0_OR_GREATER) || (__MACOS__ && !NET6_0_OR_GREATER) || HAS_SKOTTIE)
 		public void Play(double fromProgress, double toProgress, bool looped)
 		{
+#if !NET461
 			throw new NotImplementedException();
+#endif
```

Comment:
minor suggestion to avoid all those `#if` directives.

```csharp
[Conditional("NET461")]
private static void ThrowNotImplementedOnNET461()
{
    throw new NotImplementedException();
}
```

and call the method normally.

Reply:
It's actually the other way around, so Conditional can't be used, but using a single method is better for sure!